altair/utils/save.py

```diff
@@ -15,6 +16,48 @@ def write_file_or_filename(fp, content, mode="w"):
         fp.write(content)
 
 
+def set_inspect_format_argument(format, fp, inline):
+    """Inspect the format argument in the save function"""
+    if format is None:
+        if isinstance(fp, str):
+            format = fp.split(".")[-1]
+        elif isinstance(fp, pathlib.PurePath):
+            format = fp.suffix.lstrip(".")
+        else:
+            raise ValueError(
+                "must specify file format: "
+                "['png', 'svg', 'pdf', 'html', 'json', 'vega']"
```

Comment:
```suggestion
                "['png', 'svg', 'pdf', 'html', 'json', 'vega', 'vega-lite']"
```
Does it make sense to factor out this list into a variable and use it in the strings everywhere? Feels like this is repeated quite often.

This list should also be updated in the docstring of `v5/api.py:TopLevelMixin.save`